src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,49 @@
 import React from 'react';
+import { Post, User } from '../../types/types';
+import { CommentList } from '../CommentList';
+import usersFromServer from '../../api/users';
+import commentsFromServer from '../../api/comments';
 
-export const PostInfo: React.FC = () => (
```

Comment:
Function component replaced with a more verbose and functional version, nice work! 🎉